extensions/wikia/AdEngine/js/video/vastUrlBuilder.js

```diff
@@ -55,7 +56,7 @@ define('ext.wikia.adEngine.video.vastUrlBuilder', [
 		options = options || {};
 		slotParams = slotParams || {};
 
-		var correlator = Math.round(Math.random() * 10000000000),
+		var correlator = options.correlator ? options.correlator : Math.round(Math.random() * 10000000000),
```

Comment:
the correlator generated in `article-featured-video.js` is generated in the same way, why can't it be done always here?